lib/sky-pages-route-generator.js

```diff
@@ -190,7 +191,7 @@ function extractGuard(file) {
       throw new Error(`As a best practice, only export one guard per file in ${file}`);
     }
 
-    result = { path: path.resolve(file), name: match[1] };
+    result = { path: file.replace(/\\/g, '/'), name: match[1] };
```

Comment:
The `path.resolve` was basically in there as a stop-gap before that I forgot to take out. We really don't want it there as it'll make the module path absolute.. and would then break deployments too.

Additionally, this path is used for node.js module resolution would show *always* be `/` even on windows.